### ISSUE TITLE
left-align label

### ISSUE THREAD

**Issue by j4k0xb** [ts-graphviz/ts-graphviz#1127]
**Is your feature request related to a problem? Please describe.**
```js
[_.label]: "0: LdaZero\n1: Star1\n2: LdaGlobal [0], [0]",
```
`\n` justifies a line to the center. I want to justify it to the left instead.
![image](https://github.com/user-attachments/assets/ad10ed31-c432-4b57-a3d7-5a6cdc797d77)

**Describe the solution you'd like**
Support `\\l` to left-justify a line (since `\l` would be parsed as `l` by js).

**Describe alternatives you've considered**
A tagged template function to parse the raw strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#raw_strings
```js
[_.label]: escString`0: LdaZero\l1: Star1\l2: LdaGlobal [0], [0]\l`,
```

**Additional context**
https://graphviz.gitlab.io/docs/attr-types/escString/

**Comment by kamiazya** [comment]
Hi, @j4k0xb 

Thank you for your feature request and for providing detailed information along with examples. We appreciate your effort in helping us improve the library.

We understand that using `\n` in labels centers the lines by default in Graphviz, and you would like to left-align the lines using `\l`. However, since `\l` is not a recognized escape sequence in JavaScript strings, it gets parsed as a regular `l`, which poses a challenge.

Your suggestion to support `\\l` as an escape sequence to achieve left alignment is valuable. We have addressed this issue in PR #1204, which introduces proper handling of escape sequences in strings within the DOT language. This update allows you to use `\\l` in your labels to achieve left-justified lines as intended.

The PR is currently under review, and we expect to release it soon. We believe this update will provide the functionality you're looking for.

Thank you again for bringing this to our attention. Your feedback is instrumental in helping us enhance the library. If you have any more suggestions or questions, please don't hesitate to share them.

Best regards,

kamiazya

**Comment by nagasawaryoya** [comment]
@all-contributors please add @j4k0xb for bug
